packages/amplify-codegen/src/commands/add.js

```diff
@@ -20,7 +20,7 @@ const fs = require('fs-extra');
 const askForFrontend = require('../walkthrough/questions/selectFrontend');
 const askForFramework = require('../walkthrough/questions/selectFramework');
 
-const frontends = ['android', 'ios', 'javascript'];
+const frontends = ['android', 'ios', 'javascript', 'flutter'];
```

Comment:
Considering Flutter is more an SDK, might it be better to move flutter under android as a framework?

Reply:
Flutter is for multiplatform development, so flutter projects will sit on top of all frontend types.